### PR TITLE
[Editor] User feedback after user tests.

### PR DESCRIPTION
**Pull request by marcenacp**
- Remove datasets generated by OpenML converter. Context: the code was moved to https://github.com/openml/openml-croissant.
- Substitute special characters in name string.
- Add documentation on fields.
- Allow to upload a Hugging Face dataset.

### CODE REVIEW

File: editor/views/files.py [hunk 1]
```diff
@@ -202,6 +203,11 @@ def _render_resource(prefix: int, file: Resource, is_file_object: bool):
         default=file.contained_in,
         options=parent_options,
         key=key,
+        help=(
+            "FileObjects and FileSets can be nested. Specifying `Parents` allows to"
+            " nest a FileObject/FileSet within another FileObject/FileSet. An example"
+            " of this is when images (FileSet) are nested within an archive (FileSet)."
```
Comment (ccl-core) [inline editor/views/files.py:209]: FileSets?

File: editor/views/record_sets.py [hunk 2]
```diff
@@ -257,6 +259,10 @@ def _render_left_panel():
             st.checkbox(
                 "The RecordSet is an enumeration",
                 key=key,
+                help=(
+                    "Enumerations indicate that the RecordSet takes its values in a"
```
Comment (ccl-core) [inline editor/views/record_sets.py:263]: from instead of in?

File: editor/views/record_sets.py [hunk 3]
```diff
@@ -257,6 +259,10 @@ def _render_left_panel():
...
+                    " finite set. Similar to `ClassLabel` in TFDS or Hugging Face."
```
Comment (ccl-core) [inline editor/views/record_sets.py:264]: Maybe add a link to [the TFDS page](https://www.tensorflow.org/datasets/api_docs/python/tfds/features/ClassLabel) ?

File: editor/views/record_sets.py [hunk 4]
```diff
@@ -324,8 +334,14 @@ def _render_left_panel():
             )
             data_editor_key = _data_editor_key(record_set_key, record_set)
             st.markdown(
-                f"{needed_field('Fields')} (add/delete fields by directly editing the"
-                " table)"
+                needed_field("Fields"),
+                help=(
+                    "Add/delete fields by directly editing the table. Warning: the"
```
Comment (ccl-core) [inline editor/views/record_sets.py:339]: *Warning* ?

File: editor/views/record_sets.py [hunk 5]
```diff
@@ -324,8 +334,14 @@ def _render_left_panel():
...
+                    " table contains information about the fields--not the data"
+                    " directly. If you wish to embed data, select `The RecordSet is an"
+                    " enumeration` above. To edit fields details, click the button"
```
Comment (ccl-core) [inline editor/views/record_sets.py:342]: select `The RecordSet is an  enumeration` above

this sounds a bit strange to me (maybe `from above` ? or `tick the `this ...is an enumeration` box?) but as a non-native speaker I'm also not 100% sure

**Review by ccl-core — APPROVED** [review]
This is really nice!

Do we need to have someone from the openml people to LGTM too?

File: editor/views/files.py (hunk 1 above)
Comment (marcenacp) [reply editor/views/files.py:209]: Acknowledged.

Several images bundled in one FileSet.

File: editor/views/record_sets.py (hunk 2 above)
Comment (marcenacp) [reply editor/views/record_sets.py:263]: Done.

File: editor/views/record_sets.py (hunk 3 above)
Comment (marcenacp) [reply editor/views/record_sets.py:264]: Done.

File: editor/views/record_sets.py (hunk 5 above)
Comment (marcenacp) [reply editor/views/record_sets.py]: Done.

**Pull request merged**